apps/vaporgui/VContainer.cpp

```diff
@@ -0,0 +1,49 @@
+#include <QEvent>
+#include <QWidget>
+
+#include "VContainer.h"
+
+VContainer::VContainer( QWidget* containee ) 
+: QWidget() 
+{
+    QHBoxLayout* layout = new QHBoxLayout;
+    layout->setContentsMargins(
+        LEFT_MARGIN,
+        TOP_MARGIN,
+        RIGHT_MARGIN,
+        BOTTOM_MARGIN
+    );
+    setLayout( layout );
+
+    setSizePolicy( QSizePolicy::Minimum, QSizePolicy::Preferred );
+}
+
+void VContainer::Hide() {
+    QWidget* parent = parentWidget();
+    if (parent != nullptr){
+        parent->hide();
+    }
```

Comment:
Why would the Hide() function hide the parent widget?

Reply:
Legacy code that accidentally got in.  Thanks for the catch.